org.clementine_player.Clementine.yaml

```diff
@@ -0,0 +1,262 @@
+app-id: org.clementine_player.Clementine
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: clementine
+rename-desktop-file: clementine.desktop
+rename-icon: clementine
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+  - --filesystem=xdg-music
+  - --persist=.config/Clementine
```

Comment:
This should be reported upstream if not also.

Reply:
You mean that it's not using xdg-config? The weird thing is that it does for its confg, but not for its database. I'll make a bug report.

Reply:
Link?

Reply:
https://github.com/clementine-player/Clementine/issues/6122
This pretty much brings up the issue that the databases are hard-coded and cannot be changed. I could make a more specific issue if you think it's relevant.

Reply:
I can try :)